openllmtelemetry/__init__.py

```diff
@@ -1,3 +1,5 @@
 from openllmtelemetry.instrument import get_tracer, instrument
 
 __ALL__ = [instrument, get_tracer]
+
+__version__ = "0.1.0"
```

Comment:
do we need this?

https://github.com/whylabs/openllmtelemetry/blob/mainline/openllmtelemetry/version.py#L18